solutions/Cond_4/grade.js

```diff
@@ -0,0 +1,18 @@
+var grade = 200;
+
+if(grade <= 65){
+    console.log("F");
+}
+else if(grade <= 74){
+    console.log("C");
+}
+
+else if(grade <= 89){
+    console.log("B");
+}
+
+else if(grade <= 100){
+    console.log("A");
+}else{
+    console.log("Number too high.");
+}
```

Comment:
This is good as is, but do you think we can make it even better? Maybe factor out the `console.log` calls? Lemme know if you want an example.